### ISSUE TITLE
Beast, The Leatherface of Detroit

### ISSUE THREAD

**Issue by codex-krcg** [lionel-panhaleux/krcg#109]
- **text:** Remove [LSJ 20050315]. He can't equip because he can't us action cards.
- **link:** https://groups.google.com/d/msg/rec.games.trading-cards.jyhad/COcJX2hHP-E/YLUW8FaGP0kJ

**Comment by lionel-panhaleux** [comment]
Or maybe reword and add https://groups.google.com/forum/#!msg/rec.games.trading-cards.jyhad/1TXRhYopt70/OZMBb0AlfOUJ & https://groups.google.com/forum/#!msg/rec.games.trading-cards.jyhad/1TXRhYopt70/YnL8Fiaim0IJ
